Jenkinsfile_CNP

```diff
@@ -17,7 +17,7 @@ def component = "data-store-api"
 
 // Variables to switch pipeline logic and wiring per type of build
 
-def definitionStoreDevelopPr = "PR-575" // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-902" // This doesn't change frequently, but when it does, only change this value.
```

Comment:
Just before merging, let's remember to switch this back since it was a temporary change to test this PR specifically 👍